cypress/page/personal-Form.page.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable require-jsdoc */
+class PersonalFormPage {
+  private toolsQaFormURL: string;
+  private firstName: string;
+  private lastName: string;
+  private email: string;
+  private gender: string;
+  private mobileNumber: string;
+  private hobbies: string;
+  private currentAddresss: string;
+  private submit: string;
+  private submitting: string;
+  private state: string;
+  private stateOptions: string;
+  private city: string;
+
+  constructor() {
+    this.toolsQaFormURL = "https://demoqa.com/automation-practice-form";
+    this.firstName = "#firstName";
+    this.lastName = "#lastName";
+    this.email = "#userEmail";
+    this.gender = "#genterWrapper .col-md-9";
```

Comment:
if you use here, `#genterWrapper input`, then you will be able to filter by value.